docs/src/lathe/lathe-user.adoc

```diff
@@ -27,13 +27,13 @@ To set up AXIS for Lathe Mode.
 ----
 [DISPLAY]
 
-# Tell the Axis Display our machine is a lathe.
+# Tell the AXIS display (VCP) our machine is a lathe.
```

Comment:
Why "(VCP)" ?

Reply:
Isn't AXIS the virtual control panel? I found "Display" misleading.

Reply:
According to the docs, VCPs are the custom panels you can add to the GUI.
Yeah display sound not so good for this. What abut " # Tell the AXIS GUI our machine is a lathe. " ?